tools/rum-models-generator/Sources/RUMModelsGeneratorCore/RUM/RUMSwiftTypeTransformer.swift

```diff
@@ -10,7 +10,16 @@ import Foundation
 internal class RUMSwiftTypeTransformer: TypeTransformer<SwiftType> {
     /// Types which will shared between all input `types`. Sharing means detaching those types from nested declaration
     /// and putting them at the root level of the resultant `types` array, so the type can be printed without being nested.
-    private let sharedTypeNames = ["RUMConnectivity", "RUMUser", "RUMMethod", "RUMEventAttributes", "RUMCITest"]
+    private let sharedTypeNames = [
+        "RUMConnectivity",
+        "RUMUser",
+        "RUMMethod",
+        "RUMEventAttributes",
+        "RUMCITest",
+        "RUMDevice",
+        "RUMOS",
```

Comment:
I'm adding these two, to not repeat `Device` and `OS` in each event. Instead of `RUMViewEvent.Device`, `RUMErrorEvent.Device`, `RUMActionEvent.Device`, ... we will now have `RUMDevice` shared by event models.